resources/sqlsyntax.css

```diff
@@ -5,21 +5,18 @@ body{
 
 /* Keyword */
 .k{
-    /*color: rgb(148,41,41);*/
     color: #cb4b16;
     /*font-weight: bold;*/
 }
 
 /* String */
 .s{
-    /*color: rgb(20,20,232);*/
     color: #268bd2;
 }
 
 /* Tag */
 .t{
-    /*color: rgb(231,98,225);*/
-    color: #d33682;
+    color: #cc6f6f;
```

Comment:
I think we should use another color from the Solarized theme (http://ethanschoonover.com/solarized)
Right now we use "magenta" for tags, "orange" for keywords, "green" for some punctuation, and "blue" for strings.
Perhaps we should change the tags to "cyan" or "violet" (which are less aggressive than magenta)